schema.sql

```diff
@@ -0,0 +1,17 @@
+DROP DATABASE IF EXISTS sweethornbillbookstore;
+CREATE DATABASE sweethornbillbookstore;
+
+\c sweethornbillbookstore;
+
+
+CREATE TABLE books (
+  id SERIAL PRIMARY KEY,
+  title VARCHAR,
+  author VARCHAR,
+  genre VARCHAR,
+  description VARCHAR,
+  img_url VARCHAR
+);
+
+INSERT INTO books ( title, author, genre, description, img_url )
```

Comment:
so would i separate the files and then run fixture-data.sql when piping my data into the database as opposed to schema.sql?